catalog/admin/module/catalog-admin-module-sources/src/main/webapp/templates/deleteSource.handlebars

```diff
@@ -11,7 +11,10 @@
  *
  **/
  --}}
-<div class="delete-source-name">{{name}}</div>
+<div class="delete-source-name"><span style="margin-right: 2em">{{name}}</span>
```

Comment:
This currently looks like: ![screen shot 2015-11-10 at 2 29 20 pm](https://cloud.githubusercontent.com/assets/14062223/11076412/dd87f5f6-87b7-11e5-89f9-8a385890a305.png)

I also had a version where the "Delete Source" checkbox was in the same column as the other configurations under that source.

Does anyone have thoughts or preferences regarding the look?